### DISCUSSION TITLE
Renaming an instance

### DISCUSSION THREAD

**Discussion by Inc0n** [lima-vm/lima#879] [Q&A] [answered]
Would it be possible to rename some instances, e.g. archlinux to arch.

Edit: so instead of renaming an instance, one could use `export LIMA_INSTANCE=<instance name>` in their shell rc files to allow quicker access to instances other than default, which requires more a bit more effort to access via `limactl shell <instance name>`. As suggested by @jandubois in his helpful answer :)

**Comment by jandubois** [comment]
It is possible, but it seems like `archlinux` is the canonical name (e.g. the website is `http://archlinux.org`).

You can always create a symlink yourself:

```console
$ ln -s archlinux.yaml /usr/local/share/doc/lima/examples/arch.yaml
$ limactl start template://arch
[...]
```

**Reply by Inc0n** [reply]
Thank you for the solution. 

> it seems like archlinux is the canonical name (e.g. the website is http://archlinux.org).

I see I see, this need arises from saving a few key strokes, i.e. from archlinux (9) to just arch (4 characters). 

A minor convienient for a personal preference.

Using `default` template was fine, until I switched to `archlinux` recently. And now running just `lima` wouldn't work now to get the shell. 

I wonder, if `lima` the script could be modified such that it allows for passing desired instance to be shelling to, e.g. `lima archlinux`, or just simply `lima`.

Then again, this is all for but a personal preference.

**Comment by jandubois** [comment, marked as answer]
>I wonder, if lima the script could be modified such that it allows for passing desired instance to be shelling to, e.g. lima archlinux, or just simply lima.

Did you take a look at the script? 😄 

This should do what you want:

```console
$ export LIMA_INSTANCE=archlinux
$ lima
```

Put the `export` into your `.bash_profile` or `.zshrc` and you should be good to go...

**Reply by Inc0n** [reply]
Ahh yes, sorry, I did, it didn't came across to me that `export` it would make it the solution I was looking for.

Thank you!
